firebase-firestore/CHANGELOG.md

```diff
@@ -1,11 +1,9 @@
 # Unreleased (21.1.0)
-- [changed] With disk persistence enabled, query processing time for most
-  queries is now mainly dependent on the size of the result set rather than the
-  size of the locally cached collection. Queries that use the `limit()` API as
-  well as queries that have not been recently executed may not benefit from
-  this change.
-  If you are updating to v21.1.0, only queries that have been synchronized with
-  the backend at least once will see improved performance.
+- [changed] With disk persistence enabled, query processing time for repeatedly
+  executed queries is now mainly dependent on the size of the result set rather
+  than the size of the locally cached collection. Queries that use the
+  `limit()` API as well as queries that have never or have not recently been
```

Comment:
"have never or have not recently been" is probably easier to read as "have not recently (or ever) been"...

And yes, I realize now that "never" is a subset of "not recently", but thank you for being explicit for me. :-)

I'll let Gil re-tear it up now.